third_party/docfx/templates/devsite/partials/uref/class.tmpl.partial

```diff
@@ -18,7 +18,17 @@
 {{#overload}}
 <a id="{{id}}" data-uid="{{uid}}"></a>
 {{/overload}}
+{{#isPython}}

```

Comment:
Do we need this `isPython`? What happens if we don't have it? Same below.

Reply:
Nope, reverting the change. Good catch!